pkgs/by-name/ch/choose-gui/package.nix

```
@@ -0,0 +1,48 @@
+{ lib, stdenv, system, fetchFromGitHub, xcbuild, coreutils, darwin }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, xcbuild, coreutils, darwin }:
```

Hint: `stdenv.hostPlatform.system`

---

Added to the latest commit! 

Also, GitHub UI/`git` workflow question: do you have suggestions on how to deal with suggested changes? I would love to commit suggestions from GitHub's UI but don't want to add an extra commit that I have to squash later -- so I've just manually added the changes and resolved the reviews. Is there a better way to work with code suggestions that I'm not aware of?

---

I don't think there is a nice solution at GitHub UI to do that. Changes are going to be new commits. And then, I'll ask you to squash it. Ask Microsoft for a solution. :o)